src/types.ts

```diff
@@ -2137,3 +2137,18 @@ export type TruncateOptions<AttachmentType, MessageType, UserType> = {
   skip_push?: boolean;
   truncated_at?: Date;
 };
+
+export type ImportTask = {
+  created_at: string;
+  filename: string;
+  history: {
+    created_at: string;
+    next_state: string;
+    prev_state: string;
+  }[];
+  id: string;
+  state: string;
+  updated_at: string;
+  result?: UR;
+  size?: number;
+};
```

Comment:
I guess that's a bit more consistent with other types in this file:

```suggestion
export type ImportTaskHistory = {
    created_at: string;
    next_state: string;
    prev_state: string;
};

export type ImportTask = {
  created_at: string;
  filename: string;
  history: ImportTaskHistory[];
  id: string;
  state: string;
  updated_at: string;
  result?: UR;
  size?: number;
};
```

Reply:
good one